src/components/Header/Header.css

```diff
@@ -0,0 +1,28 @@
+.header {
+    display: flex;
+    flex-direction: row;
```

Comment:
I think flex defaults to row positioning. We may not need line 3.

Reply:
You right